### PR TITLE
Fix/responsive

### PR DESCRIPTION
**Pull request by LukyVj**

### CODE REVIEW

File: docgen/assets/js/sidebar.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ function toggleDocumentationSidebar() {
   function init() {
     const bodySize = document.body.clientWidth;
     if (bodySize <= 960 && sidebarNav) {
-      trigger.addEventListener('click', () => {
+      trigger.addEventListener('click', (e) => {
```
Comment (bobylito) [inline docgen/assets/js/sidebar.js]: If you don't use the event, you should remove it from the arguments of the function.

File: docgen/src/stylesheets/components/_sidebar.scss [hunk 2]
```diff
@@ -128,13 +130,14 @@ nav.sidebar {
...
   background-size: 60%, 100%;
-  z-index: 11;
+  z-index: 1000;
   border-radius: 50%;
   box-shadow: 0 6px 8px 0 rgba(0,0,0,0.14),0 1px 5px 0 rgba(0,0,0,0.12),0 3px 1px -2px rgba(0,0,0,0.2);
   transition: transform 0.2s ease, left 0.2s ease;
   transform-origin: center center;
   cursor: pointer;
   display: none;
+  pointer-events: all;
```
Comment (bobylito) [inline docgen/src/stylesheets/components/_sidebar.scss]: `all` only works on SVG elements. What's the purpose?

([mdn](https://developer.mozilla.org/en/docs/Web/CSS/pointer-events))

File: docgen/src/stylesheets/components/_sidebar.scss [hunk 3]
```diff
@@ -7,10 +7,12 @@ nav.sidebar {
   border-right: 1px solid #d8d8d8;
   overflow: hidden;
   padding-top: 120px;
+  user-select: none;
 
   .sidebar-container {
     width: 100%;
     max-width: $sidebar-width;
+    user-select: all;
```
Comment (bobylito) [inline docgen/src/stylesheets/components/_sidebar.scss]: How is `user-select` used? What's the effect on the sidebar?

**Comment by vvo** [comment]
cc @LukyVj comments awaits

File: docgen/src/stylesheets/components/_sidebar.scss (hunk 3 above)
Comment (LukyVj) [inline docgen/src/stylesheets/components/_sidebar.scss]: It's used to re-set back the `user-select: none` of above. 
<details></details>

Comment (bobylito) [inline docgen/src/stylesheets/components/_sidebar.scss]: Was not clear... Why do we set it to none in the first place?

File: docgen/src/stylesheets/components/_sidebar.scss (hunk 2 above)
Comment (bobylito) [inline docgen/src/stylesheets/components/_sidebar.scss]: OK... browsers :)

**Comment by vvo** [comment]
Will make the test pass and merge this

**Pull request merged**